x-pack/platform/plugins/shared/onechat/public/types.ts

```diff
@@ -5,13 +5,17 @@
  * 2.0.
  */
 
+import { AppMountParameters } from '@kbn/core/public';
+
 /* eslint-disable @typescript-eslint/no-empty-interface*/
 
 export interface ConfigSchema {}
 
 export interface OnechatSetupDependencies {}
 
-export interface OnechatStartDependencies {}
+export interface OnechatStartDependencies {
+  history: AppMountParameters['history'];
+}
```

Comment:
FYI: This type is used to define the list of plugin dependencies that get injected to our plugin during the start lifecycle. We can't add arbitrary properties in it.